common/build/eslint-config-fluid/src/test/file.ts

```diff
@@ -0,0 +1,6 @@
+/*!
```

Comment:
do you need two dummy files for this? can both just point to the same dummy?

Reply:
ESLint overrides are based on glob paths -- so in order to get ESLint to correctly apply the test-only overrides, the file being linted has to match the test override globs. So it either has to be a .spec.ts file or in the /test/ path. I haven't come up with a better solution :(